frappe/email/doctype/email_account/email_account.json

```diff
@@ -535,6 +535,66 @@
    "bold": 0, 
    "collapsible": 0, 
    "columns": 0, 
+   "default": "UNSEEN", 
+   "depends_on": "eval: doc.enable_incoming", 
+   "fieldname": "email_sync_option", 
+   "fieldtype": "Select", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "Email Sync Option", 
+   "length": 0, 
+   "no_copy": 0, 
+   "options": "ALL\nUNSEEN", 
+   "permlevel": 0, 
+   "precision": "", 
+   "print_hide": 0, 
+   "print_hide_if_no_value": 0, 
+   "read_only": 0, 
+   "remember_last_selected_value": 0, 
+   "report_hide": 0, 
+   "reqd": 0, 
+   "search_index": 0, 
+   "set_only_once": 0, 
+   "unique": 0
+  }, 
+  {
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "default": "250", 
+   "description": "Total number of emails to sync in initial sync process ", 
+   "fieldname": "initial_sync_count", 
+   "fieldtype": "Select", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "Initial Sync Count", 
+   "length": 0, 
+   "no_copy": 0, 
+   "options": "100\n250\n500", 
```

Comment:
@mbauskar need to have the option of 0 and all aswell though ideally have just a int value